my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/DefaultResetPasswordFormData.java

```diff
@@ -0,0 +1,37 @@
+package com.commercetools.sunrise.myaccount.authentication.recoverpassword.reset;
+
+import play.data.validation.Constraints;
+
+public class DefaultResetPasswordFormData implements ResetPasswordFormData {
+    @Constraints.MinLength(1)
+    private String newPassword;
+
+    @Constraints.MinLength(1)
+    private String confirmPassword;
+
```

Comment:
We are using play 2.5.x and they changed the validation logic a bit with 2.6. But I get your idea and will adapt it to our code.